client/v3/balancer/balancer.go

```diff
@@ -193,6 +205,11 @@ func (bb *baseBalancer) HandleResolvedAddrs(addrs []resolver.Address, err error)
 	}
 }
 
+// UpdateSubConnState implements "grpc/balancer.Balancer" interface.
+func (bb *baseBalancer) UpdateSubConnState(sc balancer.SubConn, s balancer.SubConnState) {
+	bb.HandleSubConnStateChange(sc, s.ConnectivityState)
```

Comment:
Ditto here on rename / repurpose of old code. `HandleSubConnStateChange()` is gone from the `Balancer` interface so we could just implement that here.

- https://pkg.go.dev/google.golang.org/grpc@v1.29.1/balancer#Balancer
- https://pkg.go.dev/google.golang.org/grpc@v1.35.0/balancer#Balancer